legal-api/src/legal_api/core/utils.py

```diff
@@ -49,7 +49,7 @@ def diff_dict(json1,
         if ignore_keys and key in ignore_keys:
             continue
 
-        if not json2.get(key):
+        if json2.get(key) is None and value is not None:
```

Comment:
i don't think this is correct, as this is now added to the diff ... and it is NOT different

Reply:
The previous condition `not json2.get(key)` evaluated to True when `json2.get(key)` was `False` what caused some non-changed values to the diff with wrong oldValue:
![image](https://user-images.githubusercontent.com/78114138/120021779-5c59e580-bfa0-11eb-8c7c-629c772fe904.png)



Also, when the `json2.get(key)` was `None` the condition evaluated to True and add other non-changed values to the diff:
![image](https://user-images.githubusercontent.com/78114138/120022181-e609b300-bfa0-11eb-90c1-c1d385ded83a.png)

Reply:
Please check the tests.

Reply:
thanks for combining the test, makes it clearer